sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -861,6 +863,10 @@ static String getExtractDestinationUri(String extractDestinationDir) {
       return toBuilder().setTableDescription(tableDescription).build();
     }
 
+    public Write<T> withFailedInsertRetryPolicy(InsertRetryPolicy retryPolicy) {
```

Comment:
Add a javadoc and explain that currently it works only when applying to an unbounded collection.
(the comment will need to be changed when insert method is selectable explicitly - but for now, it would feel icky to me to document that it works only when using "streaming inserts", without providing the user with a lever to use streaming inserts on a bounded collection in case they get this error)